shortcodes/class.geojson-shortcode.php

```diff
@@ -106,21 +114,31 @@ function layerStyle (feature) {
                 }      
                 function onEachFeature (feature, layer) {
                     var props = feature.properties || {},
-                        text = popup_property && props[ popup_property ] || template(popup_text, feature.properties);
+                        text = popup_property && props[ popup_property ] || template(popup_text, feature.properties),
+                        species = feature.properties.name,
```

Comment:
I don't think you should use species as a var name, because if you wanted to push this code upstream it would need to be something generic, like `name`.

Reply:
Changed to featureName